arabot/modules/serverinfo.py

```diff
@@ -0,0 +1,219 @@
+        assets = []
+        if isinstance(guild, disnake.Guild):
+            if guild.vanity_url_code:
+                assets.append(f"[Vanity invite](https://discord.gg/{guild.vanity_url_code})")
+            if guild.banner:
+                assets.append(f"[Banner]({guild.banner.maxres})")
+        if guild.splash:
+            assets.append(f"[Invitation image]({guild.splash.maxres})")
+        if guild.discovery_splash:
+            assets.append(f"[Discovery image]({guild.discovery_splash.maxres})")
+        embed.description = " • ".join(assets)
+        if guild.description:
+            embed.description += "\n" + guild.description
+
+    @staticmethod
+    def _set_footer(embed: disnake.Embed, guild: disnake.Guild | disnake.GuildPreview):
+        shown_features = {"VERIFIED", "PARTNERED"}
+        if features := [feature for feature in guild.features if feature in shown_features]:
+            embed.set_footer(text=", ".join(features).capitalize() + " server")
+
+    @staticmethod
+    def _set_field_general_info(
+        field_values: list, guild: disnake.Guild, current_guild: disnake.Guild
+    ) -> None:
+        if not isinstance(guild, disnake.Guild):
+            return
+
+        owner = f"<@{guild.owner_id}>"
+        if guild != current_guild and guild.owner:
+            owner = mono(guild.owner)
+        field_values.append(f"Owner: {owner}")
+        field_values.append(f"Created on {format_dt(guild.created_at, 'D')}")
+        field_values.append(f"Locale: {str(guild.preferred_locale).upper()}")
+        field_values.append(f"Boost level: {guild.premium_tier}")
+        field_values.append(f"Upload limit: {guild.filesize_limit >> 20} MB")  # I know this is MiB
+        match guild.nsfw_level:
+            case disnake.NSFWLevel.safe:
+                field_values.append("NSFW level: safe")
+            case disnake.NSFWLevel.age_restricted:
+                field_values.append("May contain NSFW content")
+            case disnake.NSFWLevel.explicit:
+                field_values.append("Contains NSFW content")
+
+    @staticmethod
+    def _set_field_channels(field_values: list, guild: disnake.Guild) -> None:
+        if not isinstance(guild, disnake.Guild):
+            return
+        if guild.categories:
+            field_values.append(f"Categories: {len(guild.categories)}")
+        if guild.text_channels:
+            field_values.append(f"Text: {len(guild.text_channels)}")
+        if guild.threads:
+            field_values.append(f"Threads: {len(guild.threads)}")
+        if guild.voice_channels:
+            field_values.append(f"Voice: {len(guild.voice_channels)}")
+        if guild.stage_channels:
+            field_values.append(f"Stage: {len(guild.stage_channels)}")
+        if guild.forum_channels:
+            field_values.append(f"Forum: {len(guild.forum_channels)}")
+        # if guild.rules_channel:
+        #     field_values.append(f"Rules: {guild.rules_channel.mention}")
```

Comment:
I strongly suggest against committing commented out code, unless there is a good reason *and* a work item in the backlog for it.